LSPS1/README.md

```diff
@@ -234,10 +234,10 @@ The client MUST check if [option_support_large_channel](https://bitcoinops.org/e
 | Code   | Message         | Data | Description |
 | ----   | -------         | ----------- | ---- |
 | -32602 | Invalid params  | {"property": %invalid_property%, "message": %human_message% }    | Invalid method parameter(s). |
-| 1000   | Option mismatch |  {"property": %option_mismatch_property%, "message": %human_message% }   | The order doesnt match the options defined in `lsps1.get_info.options`. |
-| 1001   | Client rejected |  {"message": %human_message% }   | The LSP rejected the client. |
+| 100    | Option mismatch |  {"property": %option_mismatch_property%, "message": %human_message% }   | The order doesnt match the options defined in `lsps1.get_info.options`. |
+| 101    | Client rejected |  {"message": %human_message% }   | The LSP rejected the client. |
```

Comment:
Should we move this to LSPS0?

I think this option is useful in every LSPS. 
Especially LSPS2 where a server might have opened multiple channels for which the client never paid.

Reply:
We could add common errors to the LSPS0 Common Schemas doc.